### DISCUSSION TITLE
Can not install this at all.

### DISCUSSION THREAD

**Discussion by DominikCGI** [oobabooga/text-generation-webui#2764] [Q&A]
First of all there is no file that is just called "start" like the description tries to tell us. Either it is missing or has been replaced by a file with a different name. By running "start_windows.bat" which is my next best guess it stucks on 32% during the git 2.40.1 download. Effectifly blocking all progress. Is there anyway to fix this faulty installation process. There clearly are issues in the bat file or the right file is not included. Please help!

**Comment by viperwasp** [comment]
I am not a pro. But I'm assuming you have windows 10 or 11? You downloaded the one click installer found on this page. 
https://github.com/oobabooga/text-generation-webui

You are right you click "start_windows.bat" 
It should run commands until you get an option to choose from like Nvidia etc. 

What GPU do you use Nvidia, AMD, ARC? 
If you use Nvida select that option. I forget what the other options even are since I use nvidia. 

If your getting stuck before it gives you pop up to choose Nvidia. Than I'm not sure. Never had that problem. If your getting any red text or errors maybe copy it here? Also make sure your not clicking on the CMD window as it can pause it. Sometimes the setup does take a while but if it's more than 5 minutes or if it says error or closes on it's own anything like that is an issue. 

So copy and erros or copy and paste the last few lines were it got stuck. So we can see were it's stuck. YOu said 32% during the git 2.40.1 download. That might be enough for some users to know how to fix. I don't have any ideas. Some possible solutions could be to like install git yourself manually? But before that lets see what other people have to say. Or maybe if you copy and paste over more information it might help.
